### DISCUSSION TITLE
Reorder items in shopping list

### DISCUSSION THREAD

**Discussion by gringogriot** [mealie-recipes/mealie#4795] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

The list of items in a shopping list is static and not changeable.

### Please provide a concise description of the feature that would resolve your issue.

Some form of (JS-based?) click-and-drag function allowing the user to re-order shopping list items.

### Please consider and list out some caveats or tradeoffs made in your design decision

I'm not sure there are any! I'd love to be able to offer assistance in implementing this, but I'm afraid my coding skills aren't sufficient for that.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by ChipSkylark37** [comment]
Unless you are talking about something else - There is a way to do this via drag-and-drop (both Mobile / Website).

https://snipboard.io/KCiDId.jpg

Do you not have this grid icon to the right that lets you re-order the list?

**Reply by gringogriot** [reply]
So there is! I'd initially assumed that those grid icons were there for re-ordering items, but when I clicked them, I was shown the "Edit / Delete" context menu. After a refresh (and being a little more deliberate with the awful touchpad on the laptop I'm using), I see the re-order function. Wonderful! 

Closing this discussion as PEBKAC. Thank you for a wonderful app!

**Reply by ChipSkylark37** [reply]
PEBKAC - I haven't seen that in a while! Glad to hear it's working. 

I re-organize my list constantly to help me keep a single shopping list but sort by how I expect to shop and what store may have what. Or simply to just put "Eh, pickup whenever" at the bottom of my list and ones I really need / want at the top.

Cheers!
